plugins/inventory/lxd.py

```diff
@@ -566,6 +571,7 @@ def extract_information_from_instance_configs(self):
             self.data['inventory'] = {}
 
         for instance_name in self.data['instances']:
+            self._set_data_entry(instance_name, 'os', self._get_data_entry('instances/{0}/instances/metadata/config/image.os'.format(instance_name)))
```

Comment:
Why ad this line? The next lines already do this, don't they?

Reply:
right. I think that was my editor.